explorer/websockethandlers.go

```diff
@@ -106,20 +106,29 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 					webData.Message = string(msg)
 
 				case "getticketpooldata":
-					// TODO: Implement a cache to control websocket connections
-					// meant to serve the same data.
-					cData, gData, err := exp.explorerSource.TicketPoolVisualization(
-						dbtypes.ChartGroupingFromStr(msg.Message),
-					)
-					if err != nil {
-						if strings.HasPrefix(err.Error(), "unknown interval") {
-							log.Debugf("Invalid ticket pool interval provided via getticketpooldata: %s", msg.Message)
-							webData.Message = "Error: " + err.Error()
+					var err error
```

Comment:
Might as well limit the scope of this.  Putting it after `if !ok {` would seem to be fine.

Reply:
ok. Noted